AsyncDisplayKit/Layout/ASRelativeLayoutSpec.h

```diff
@@ -15,21 +15,21 @@
 /** How the child is positioned within the spec. */
 typedef NS_OPTIONS(NSUInteger, ASRelativeLayoutSpecPosition) {
     /** The child is positioned at point 0 relatively to the layout axis (ie left / top most) */
-    ASRelativeLayoutSpecPositionStart = 0,
+    ASRelativeLayoutSpecPositionStart = 1 << 0,
     /** The child is centered along the specified axis */
-    ASRelativeLayoutSpecPositionCenter = 1 << 0,
+    ASRelativeLayoutSpecPositionCenter = 1 << 1,
     /** The child is positioned at the maximum point of the layout axis (ie right / bottom most) */
-    ASRelativeLayoutSpecPositionEnd = 1 << 1,
+    ASRelativeLayoutSpecPositionEnd = 1 << 2,
 };
 
 /** How much space the spec will take up. */
 typedef NS_OPTIONS(NSUInteger, ASRelativeLayoutSpecSizingOption) {
     /** The spec will take up the maximum size possible */
-    ASRelativeLayoutSpecSizingOptionDefault,
+    ASRelativeLayoutSpecSizingOptionDefault = 1 << 0,
```

Comment:
Maybe this is a bad idea, but couldn't this also be defined as an NS_ENUM  
```
typedef NS_ENUM(NSUInteger, ASRelativeLayoutSpecSizing) {
  ASRelativeLayoutSpecSizingDefault               = 0, 
  ASRelativeLayoutSpecSizingMinimumWidth  = 1,   
  ASRelativeLayoutSpecSizingMinimumHeight = 2, 
  ASRelativeLayoutSpecSizingMinimumSize     = 3,
};
```
Though I'm sure that would need some changes in ASRelativeSpec to make work.